iroha/tests/test_config.json

```diff
@@ -8,7 +8,7 @@
     "COMMIT_TIME_MS": 1000,
     "TX_RECEIPT_TIME_MS": 100,
     "MAX_FAULTY_PEERS": 1,
-    "TRUSTED_PEERS": [
+    "IROHA_TRUSTED_PEERS": [
```

Comment:
`IROHA_TRUSTED_PEERS` is the name of the environment variable, but the name of the field is just `TRUSTED_PEERS`. So as far as I remember the config is correct here.